From d2bc527c12a74675d67ba5fbf9de0d014af929cb Mon Sep 17 00:00:00 2001
From: Dan Radez <dradez@redhat.com>
Date: Tue, 12 Apr 2022 09:40:46 -0400
Subject: [PATCH] Replace readfp with read_file

Getting Deprecation Warning in python 3.11:
DeprecationWarning: This method will be removed in Python 3.12.
Use 'parser.read_file()' instead.

https://github.com/python/cpython/commit/e8659b47dece5a272111c0af5e340c364a9f807b#diff-ca80d44aeeadd5e4966195b99aff106abb3e7b0b5356b7499125ee021bd84777R757

If I'm reading this correct read_file has been around since Python 2.7
https://github.com/python/cpython/commit/a492362f9a2a44e411147fd7b2886466bb0bb17f
---
 cherrypy/lib/reprconf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cherrypy/lib/reprconf.py b/cherrypy/lib/reprconf.py
index 76381d7b7..536b94173 100644
--- a/cherrypy/lib/reprconf.py
+++ b/cherrypy/lib/reprconf.py
@@ -188,7 +188,7 @@ def as_dict(self, raw=False, vars=None):
 
     def dict_from_file(self, file):
         if hasattr(file, 'read'):
-            self.readfp(file)
+            self.read_file(file)
         else:
             self.read(file)
         return self.as_dict()
